apps/src/templates/certificates/SocialShare.jsx

```diff
@@ -46,6 +46,7 @@ export default class SocialShare extends Component {
             <button
               type="button"
               style={{background: color.facebook_blue, ...styles.shareButton}}
+              onClick={e => e.preventDefault()}
```

Comment:
Once I fixed the URL issues, the links were opening twice for me. Adding this prevented that, so I just went with it. Let me know if I should investigate further!